src/coreclr/src/jit/emitarm64.cpp

```diff
@@ -10763,10 +10997,9 @@ void emitter::emitDispLSExtendOpts(insOpts opt)
         assert(!"Bad value");
 }
 
-/*****************************************************************************
- *
- *  Display a register
- */
+//------------------------------------------------------------------------
+// emitDispReg: Display a general-purpose register name or SIMD and floating-point scalar register name
```

Comment:
nit: If we are updating these, we should likely do it fully and also document the parameters, etc

Reply:
agree. will do